README.md

```diff
@@ -37,18 +37,94 @@ The latest version from the release branch 'master' is automatically deployed to
 
 $ `git clone https://github.com/ita-social-projects/ZeroWaste.git`
 
-## Setup
+## Local setup
+First of all you need rvm to setup project. For the operating system Windows the optimal solution is to use <a href="https://docs.microsoft.com/en-us/windows/wsl/">WSL</a>.
 
 $ `bin/setup`
-or 
-$ `bundle install` 
+or
+$ `bundle install`
+
+<b>PostgreSQL</b>
+
+<a href="https://www.postgresql.org/download/">Install PostgreSQL</a> for your operating system or subsystem.
+You can familiarize yourself with <a href="https://www.postgresql.org/docs/">PostgreSQL documentation</a>.
+
+In your local machine in cloned project in config folder rename database.yml.sample to database.yml. Make sure that the user and password match the data in this file. Port may be changed.
+
+<b>pg gem</b>
+
+Under certain circumstances bundle can do not install pg.
+
+To install manually:
+$ `sudo apt-get install libpq-dev`
+then
+$ `gem install pg`
+
+<b>Database configure</b>
+
+For correct operation of the migration, you need to rename the migration file `20220123171144_create_versions.rb` so that it is processed first.
+
+To create the necessary databases and update them:
+
+$ `rake db:create`
+then
+$ `rake db:migrate`
+
+<b>Redis</b>
+
+You need Redis for correct work.
+<a href="https://redis.io/docs/getting-started/">Install Redis</a> for your operating system or subsystem. You can familiarize yourself with
+<a href="https://redis.io/docs//">Redis documentation</a>.
+
+Installation for Ubuntu:
+
+$ `curl -fsSL https://packages.redis.io/gpg | sudo gpg --dearmor -o /usr/share/keyrings/redis-archive-keyring.gpg`
+
+$ `echo "deb [signed-by=/usr/share/keyrings/redis-archive-keyring.gpg] https://packages.redis.io/deb $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/redis.list`
+
+$ `sudo apt-get update`
+$ `sudo apt-get install redis`
+
+<b>npm and Node.js</b>
+
+Also you need Node.js.
+<a href="https://nodejs.org/en/download/">Install npm and Node.js</a> for your operating system or subsystem. You can familiarize yourself with
+<a href="https://nodejs.org/en/about/">npm and Node.js documentation</a>
+
+<b>yarn</b>
+
+You can read more about yarn there:
+<a href="https://classic.yarnpkg.com/lang/en/docs/">yarn documentation</a>.
+
+Installation:
+$ `npm install --global yarn`
+
+<b>Webpacker</b>
+
+To prevent an error when starting the server install webpacker. You can read more about webpacker there:
+<a href="https://guides.rubyonrails.org/webpacker.html">Webpacker documentation</a>.
+
+Installation:
+$ `yarn add @rails/webpacker`
+$ `bundle update webpacker`
+
+<b>Sidekiq</b>
+
+Simple, efficient background processing for Ruby. You can read more about sidekiq there:
+<a href="https://github.com/mperham/sidekiq">Sidekiq documentation</a>.
+
+Installation:
+$ `bundle add sidekiq`
 
 ## How to run local
```

Comment:
In some systems, after restarting them, the postgresql server remains disabled, perhaps at the first start you should enter "sudo service postgresql start"